src/neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -89,7 +89,7 @@ private void OnMessage(Message msg)
                     OnAddrMessageReceived((AddrPayload)msg.Payload);
                     break;
                 case MessageCommand.Block:
-                    OnInventoryReceived((Block)msg.Payload);
+                    OnBlockReceived((Block)msg.Payload);
```

Comment:
I think it should be handled in `OnInventoryReceived()`.

Reply:
Done